exchanges/gateio/gateio.go

```diff
@@ -106,16 +106,16 @@ func (g *Gateio) GetLatestSpotPrice(symbol string) (float64, error) {
 // GetTicker returns a ticker for the supplied symbol
 // updated every 10 seconds
 func (g *Gateio) GetTicker(symbol string) (TickerResponse, error) {
-	urlPath := fmt.Sprintf("%s/%s/%s/%s", g.API.Endpoints.URLSecondary, gateioAPIVersion, gateioTicker, symbol)
+	urlPath := fmt.Sprintf("/%s/%s/%s", gateioAPIVersion, gateioTicker, symbol)
 	var res TickerResponse
-	return res, g.SendHTTPRequest(urlPath, &res)
+	return res, g.SendHTTPRequest(exchange.WebsocketSpotSupplementary, urlPath, &res)
```

Comment:
I do not understand why you're using `WebsocketSpotSupplementary` for REST requests where `WebsocketSpotSupplementary` equals the REST endpoint

Reply:
Yeah that was a typo. Was supposed to be RestSpotSupplementary. Fixed now